logitech_r400/config/config.yaml

```diff
@@ -0,0 +1,42 @@
+version_0:
```

Comment:
If you are using names like version_0, version_1 and so on, it means that this should be a vector, not a dictionary.